docs/README.md

```diff
@@ -61,6 +61,9 @@ In addition to the [literal](../examples/pipelines/fl_cross_silo_literal/) and [
 ### Pneumonia detection from chest radiographs
 In this example, we train a model to detect pneumonia from chest radiographs. The model is trained on a dataset from the [NIH Chest X-ray dataset](https://www.kaggle.com/nih-chest-xrays/data). This example is adapted from [that solution](https://github.com/Azure/medical-imaging/tree/main/federated-learning) by Harmke Alkemade _et al._ See [here](./real-world-examples/pneumonia.md) for detailed instructions on how to run this example.
 
+### Named Entity Recognition using MultiNERD dataset
```

Comment:
@jfomhover  Currently all the 3 real world fl examples all only provide literal submission script (i.e., not using pipeline subgraph, not using fl-factory). Do we want to provide submission script that leverage pipeline subgraph and/or fl factory here? Or you prefer to only keep the literal submission script for these 3 real world examples for now?

Reply:
let's stick to literal for now